pmd-java/src/test/resources/net/sourceforge/pmd/ast/ParserCornerCases18.java

```diff
@@ -165,6 +165,27 @@ public void lambdaWithPropertyAssignment() {
         }
     }
 
+    public List<@AnnotatedUsage ?> testWildCardWithAnnotation() {
+    	return null;
+    }
+    
+	public Object @Nullable [] testAnnotationsToArrayElements() {
+		return null;
+	}
+	
+	private byte @Nullable [] getBytes(){
+		return null;
+	}
+	
+	public static <T extends @NonNull Enum<T>> getEnum() {
```

Comment:
this declaration is missing the return type, this should fail to parse... you probably meant to write: `public static <T extends @NonNull Enum<T>> T getEnum() {`